lib/chef/resource/ohai.rb

```diff
@@ -90,6 +90,7 @@ def load_current_resource
           ohai_plugin_path = Chef::Config[:ohai_segment_plugin_path]
           if ohai_plugin_path && Dir.exist?(ohai_plugin_path) && !Dir.empty?(ohai_plugin_path)
             # Configure Ohai to load plugins from the cookbook segment path
+            ohai.config[:plugin_path] ||= []
```

Comment:
FYI, as discussed through Slack, this still creates errors. When loading custom plugins, and thus entering this code, the default plugins aren't loaded anymore.

I was able to fix it by using the following:
```suggestion
           ohai.config[:plugin_path] ||= ::Ohai.config[:plugin_path]
```